PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Status
+**READY/IN DEVELOPMENT**
+
+## Related Issue
+[link]()
+
+## Description
+* A few sentences describing the overall goals of the pull request's commits.
+* List general components of the application that this PR will affect.
+
+## WIKI Updates
+* [link]()
```

Comment:
Let's just put one line of links here so we don't have to delete a bunch of lines for each PR